docs/source/users-guide.rst

```diff
@@ -451,7 +451,7 @@ Elliptic Curve (ECDSA) offers the following key size options:
 +--------+--------------+-----------------------+
 | 384    | secp384r1    | ecdsa-with-SHA384     |
 +--------+--------------+-----------------------+
-| 521    | secp521r1    | ecdsa-with-SHA512     |
```

Comment:
Although it looks strange, 521 is a real curve. @ale-linux can you confirm it is supposed to be 521?

Reply:
The curve is indeed called variations of P and 521 (which is the bitlength of the prime group order), not 512. That said: the curve is being phased out and we in fact no longer support it, so we can just go ahead and remove this row.

Reply:
@denyeart that line is removed from the document